spec/index.bs

```diff
@@ -7960,6 +7960,25 @@ dictionary GPURenderPassColorAttachment {
         render pass. If not [=map/exist|provided=] defaults to `{r: 0, g: 0, b: 0, a: 0}`. Ignored
         if {{GPURenderPassColorAttachment/loadOp}} is not {{GPULoadOp/"clear"}}.
 
+        The members of {{GPURenderPassColorAttachment/clearValue}} are all double values, so
+        they will first be converted to the fully qualified format type of
+        {{GPURenderPassColorAttachment/view}} before being set as the clear value of
+        {{GPURenderPassColorAttachment/view}}.
+
+        Let |colorAttachmentFormat| be
+        {{GPURenderPassColorAttachment/view}}.{{GPUTextureView/[[descriptor]]}}.{{GPUTextureViewDescriptor/format}}.
+        - If the components of |colorAttachmentFormat| are floating point values, the members of
+            {{GPURenderPassColorAttachment/clearValue}} will be converted into floating point
+            values.
+        - If the components of |colorAttachmentFormat| are signed or unsigned normalized values,
+            the members of {{GPURenderPassColorAttachment/clearValue}} will first be clamped
+            into the range [0, 1], then converted into floating point values, and finally
```

Comment:
intermediate conversion to "floating point value" not necessary because it's already a floating-point value

Reply:
I'd like to convert the double values to float first because on D3D12 the clear value can only be float, and I think we can align the behaviors on all the backends if we always use float.

Reply:
ah, in that case we would need to specify the intermediate format is 32-bit/single-precision float (as you now have). But I don't think we should spec the extra conversion as it could result in double-rounding. Instead I would specify the conversion directly from float64 to the format type, and make D3D12 implementations responsible for picking the float32 value that results in the proper end result (or [polyfilling](https://github.com/gpuweb/gpuweb/issues/1085#issuecomment-703858922) when necessary).

Reply:
I've updated my latest PR with your suggestions here. PTAL, thanks!